moldesign/_tests/test_pdbfixer_xface.py

```diff
@@ -91,6 +92,22 @@ def test_mutation_nomenclature_string_only(pdb3aid):
     assert len(mut25) == 2
     assert [r.resname for r in mut25] == ['MET', 'MET']
 
+def test_mutation_topology():
```

Comment:
Pytest style notes: you'll want to use a [fixture](https://docs.pytest.org/en/latest/fixture.html) to pass in the 1YU8 `Molecule` object here. We already have one ready to go - use `test_mutation_topology(pdb1yu8)`